src/admin-events/admin-events.spec.js

```diff
@@ -0,0 +1,106 @@
+const ethers = require('ethers');
+const { getAddress } = require('@uma/contracts-node');
+
+// Load Config files
+const { TransactionEvent } = require('forta-agent');
+
+const chainId = 1;
+const votingAddressPromise = getAddress('Voting', chainId);
+
+const { createAlert, handleTransaction } = require('./admin-events');
```

Comment:
I'd group all of the require statements together at the top of the file

Reply:
Done